RELEASE.md

```diff
@@ -1,33 +1,54 @@
-## Making a release of the Checker Framework Gradle plugin
+## Making and merging pull requests
 
-This document describes how to make a release of the Checker Framework
-Gradle plugin, which this repository contains.
+The Checker Framework Gradle plugin (which this repository contains)
+uses a continuous-delivery style of releases: any time that the
+behavior of the plugin changes, a new release is cut.
 
-### Prequisites
+### Making a pull request
+
+All pull requests should be made from a feature branch. Never push
+directly to `master`.
+
+When you make a pull request that will change any behavior
+of the plugin (including bug fixes and new features), update the version string on 
+your feature branch using these steps:
+1. Make the changes you wish to merge into the plugin.
+2. Choose a new version string. Please try to respect
+[semantic versioning](https://semver.org/). Do not augment the major
+version without explicit approval from all the maintainers.
+3. Change the version of the plugin. Search for the current plugin version
```

Comment:
My comment has not been addressed.  The text "Change the version of the plugin." applies to both points 2 and 3 (but not to points 1 and 4), so they should be subbullets of a single point.  If you don't want to do that, then change the confusing text for point 3 so that it is clear that it is merely an editing operation about the files in the repository.